meshpy/four_c/solid_shell_thickness_direction.py

```diff
@@ -204,7 +207,10 @@ def set_solid_shell_thickness_direction(
     for element in elements:
         is_hex8 = isinstance(element, VolumeHEX8)
         if is_hex8:
-            is_solid_shell = "SOLIDSH8" in element.dat_pre_nodes
+            is_solid_shell = (
+                hasattr(element, "dat_pre_nodes")
```

Comment:
Why do we need this? Is this added automatically?

Reply:
Otherwise mypy would have thrown the following error:

```
Run mypy (static type checker for python)............................................Failed
- hook id: mypy
- exit code: 1

meshpy/four_c/solid_shell_thickness_direction.py:210: error: "Element" has no attribute "dat_pre_nodes"  [attr-defined]
Found 1 error in 1 file (checked 67 source files)
```